vm-to-vm-bandwidth-meter/azuredeploy.json

```diff
@@ -0,0 +1,611 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "probeVmSize": {
```

Comment:
It is Standard_A2 by default, but to make the tool/template really usable the intention was to expose it to be able to measure bandwidth any-size-to-any-size (all possible regions). There is a note in the readme however, that the size should be available for the region. Would appreciate if we can leave this easily changeable.